.github/workflows/commit.yaml

```diff
@@ -86,7 +85,6 @@ jobs:
       matrix:
         go-version:
         - "1.17"  # == ${{ env.GO_VERSION }} because matrix cannot expand env variables
-        - "1.16"  # temporarily support go 1.16 per #37
```

Comment:
We can add 1.18 once it is out of beta. Typically github adds it a week or so after GA. Then, you can look for it like so: https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#go